CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [v0.5.1](https://github.com/docker/notary/releases/tag/v0.5.1) 11/14/2016
 + If no `records` parameter is provided in a request for the changefeed, a default value of 100 is used.
++ Fix build tags for static notary client binaries in linux
```

Comment:
non-blocking nit: can we add this PR number to the changelog?  I think we forgot to do it for the previous entry as well